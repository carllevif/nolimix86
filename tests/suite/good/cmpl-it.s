  .text
cmpl $0, %t0
