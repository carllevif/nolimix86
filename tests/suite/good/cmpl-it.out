  .text
cmp $0, %t0
